src/NodeManager.php

```diff
@@ -80,6 +82,7 @@ public function generateMultiple(array $ids): array
      */
     private function generateBuffer(NodeModel $nodeModel): string
     {
+        $nodeTpl = 'node_default';
```

Comment:
I guess we don't need this variable, do we? We can just use it on line 99 directly :)